server/engine/src/main/java/org/eclipse/lsp/cobol/implicitDialects/sql/Db2SqlVisitorHelper.java

```diff
@@ -150,4 +155,58 @@ public static Position getAdjustedEndPosition(
     }
     return end;
   }
+
+  public static ExecSqlWheneverNode.WheneverConditionType getConditionType(Db2SqlParser.Dbs_wheneverContext ctx) {
```

Comment:
Is there a particular reason to move the functions here and make the code changes order of magnitude bigger then they need to be?

Reply:
The original visitor class is too big already and contains helper functionality, so make sense to clean visitor up a little bit to simplify main logic understanding